### ISSUE TITLE
New cards not in random order.

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#2829]
Originally reported on Google Code with ID 1927

```
When you study a deck that has sibling cards, and you set "New card order" to "New cards
in random order", Ankidroid does not put the CARDS in random order. Instead, it puts
the NOTES in random order, instead of the CARDS.

For example, this is what AnkiDroid considers to be "New cards in random order" at
the moment:

[Note_003] Card 1:   Q: Dollars |  A: $$$
[Note_003] Card 2:   Q: $$$     |  A: Dollars
[Note_001] Card 1:   Q: Pounds  |  A: £££
[Note_001] Card 2:   Q: £££     |  A: Pounds
[Note_002] Card 1:   Q: Yen     |  A: ¥¥¥
[Note_002] Card 2:   Q: ¥¥¥     |  A: Yen

As you can see, sibling cards ALWAYS follow one another. Only the note order has been
randomized.

What it should be showing (as in Anki desktop) is something like this:

[Note_003] Card 1:   Q: Dollars |  A: $$$
[Note_001] Card 2:   Q: £££     |  A: Pounds
[Note_002] Card 2:   Q: ¥¥¥     |  A: Yen
[Note_001] Card 1:   Q: Pounds  |  A: £££
[Note_003] Card 2:   Q: $$$     |  A: Dollars
[Note_002] Card 1:   Q: Yen     |  A: ¥¥¥

In other words, it isn't scrambling up the CARDS correctly. Only the NOTES.



>Does it happen again every time you repeat the steps above?
I clean-reinstalled AnkiDroid several times to make sure. It's a problem with the software.

>What version of AnkiDroid?
2.0.2

>What version of Android?
2.3.7
```

Reported by `tiffany.f23` on 2013-12-20 09:11:42

**Comment by mikehardy** [comment]
I use this setting personally (randomize new cards) and in 2.8.4 I don't see this
